exp/api/v1beta1/awsmachinepool_types.go

```diff
@@ -76,6 +76,13 @@ type AWSMachinePoolSpec struct {
 	// +optional
 	DefaultCoolDown metav1.Duration `json:"defaultCoolDown,omitempty"`
 
+	// The amount of time, in seconds, until a new instance is considered to
```

Comment:
Its not required to add it in v1beta1 API, we have to fix the failing fuzz tests, PTAL at some other pRs where we are adding a new field and handling the tests.

Reply:
I've removed it and fixed the test. Can you please check if I did it correctly?